### DISCUSSION TITLE
EQ2 Check It writing

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#323] [Ideas]
EQ2 is Applications of Linear Equations, so will need lots of different prompts for the questions. Starting this discussion to hold space to write a bunch of those. 

Types to consider: 

Distance/rate time types:
- Objects moving toward each other
- Objects moving away from each other
- Objects covering the same distance, but at different rates (like go somewhere and come back, but one way had a lot of traffic)
- Objects going same direction, when does one catch up to the other. (Like a race where one runner comes from behind. At what time do they pass the other runner.)

Percent mixtures

- Investments earning at different rates
- Chemistry problem where one solution is x% alcohol and the other is another amount and the mixture is some % between
- % defective of a product from a certain manufacturer, combined with % defective from a different plant for that manufacturer

Value mixtures:	

- Bulk candy/coffee/etc buying, certain prices/lb
- Target sells Hot Wheels in three-packs and five-packs. On a certain day there were x three packs sold and y five packs. In total, z individual cars were sold. How many five packs?
- Child ticket costs ones amount, adult ticket another amount. How many child tickets sold.

**Comment by AbbyANoble** [comment]
**Objects moving toward each other**

- Two trains are 300 miles apart and heading toward each other. One is traveling at 75 mph and the other at 95 mph. After how much time will they meet?
- Two trains are 360 miles apart and traveling toward each other. One train is traveling 10mph faster than the other. If they meet after 2 hours, how fast is each train moving?

Replace train with other vehicles (bikes, spaceships, cars, people walking, etc). Adjust values and units to make sense for the situation.

**Comment by AbbyANoble** [comment]
**Objects moving away from each other**

- Two trains leave a station at the same time, one traveling north and the other south. One is traveling at 75 mph and the other at 95 mph. After how much time will they be 300 miles apart?
- Two trains leave a station at the same time, one traveling north and the other south. One train is traveling 10mph faster than the other. If they are 360 miles apart after 2 hours, how fast is each train moving?

Replace train with other vehicles (bikes, spaceships, cars, people walking, etc). Adjust values and units to make sense for the situation.

**Reply by tdegeorge** [reply]
I do feel like the train examples are pretty typical in this section, but not always something students care about. Should we also include more of the mixture problems with candy/food/etc.?

**Reply by tdegeorge** [reply]
Like maybe a candy example?
In Natasha’s candy shop, chocolate, which sells for $4 a kilogram, is mixed with nuts, which are sold for $2.50 a kilogram. Chocolate and nuts are combined to form a chocolate-nut candy, which sells for  $3.50 a kilogram. How much of each are used to make 30 kilograms of the mixture?

**Reply by tdegeorge** [reply]
Or a smoothie example?
A smoothie shop sells pureed strawberries for $3.50 a pound and pureed bananas for $1.50 a pound. If you ask nicely, they'll also sell a 4 pound strawberry banana puree for $2.25 per pound. How much of the strawberry and banana purees are used to make the strawberry banana puree?

**Reply by AbbyANoble** [reply]
Yeah, those would be in the value/percent mixture categories. I was just making one post for each type of these problems.

**Reply by AbbyANoble** [reply]
I added your ideas to another post below! Trying to make it easy-ish on whoever (ahem, @StevenClontz ) writes these Check-Its. :)

**Comment by siwelwerd** [comment]
Trying to think of some *real* things and not the contrived examples that I am sure we can find plenty of.

**Percent mixtures**
- Hamburger blend made from X% of Ground Chuck (20+randint(-4,4)%fat) and (100-x)% of brisket (10+randint(-4,4)% fat)--what is the fat percentage?
- ABV of a cocktail made from different ABV spirits (or maybe we don't want to go there...)
- Blending gasoline (usually gas stations have tanks full of two octanes, and blend them to get the mid grade)

**Reply by tdegeorge** [reply]
I think the food and gas examples make sense. I would probably stay away from the cocktails though :-p

**Comment by tdegeorge** [comment]
Are we saying that we will have multiple tasks? Or are these just examples of the types of problems students will have to solve?

Also, do we want to include some of the geometry questions you found in MyOpenMath? I like the area/perimeter ones...

**Reply by AbbyANoble** [reply]
Probably just one task. But make a lot of outlines and choose one from there. Like https://teambasedinquirylearning.github.io/calculus/2024e/exercises/#/bank/AD8/5/

**Comment by AbbyANoble** [comment]
**Value Mixtures**

- In Natasha’s candy shop, chocolate, which sells for $4 a kilogram, is mixed with nuts, which are sold for $2.50 a kilogram. Chocolate and nuts are combined to form a chocolate-nut candy, which sells for $3.50 a kilogram. How much of each are used to make 30 kilograms of the mixture?
- A smoothie shop sells pureed strawberries for $3.50 a pound and pureed bananas for $1.50 a pound. If you ask nicely, they'll also sell a 4 pound strawberry banana puree for $2.25 per pound. How much of the strawberry and banana purees are used to make the strawberry banana puree?
